pos_report_sessionsummary/README.rst

```diff
@@ -0,0 +1,32 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
```

Comment:
Use the new OCA readme system: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst